src/App.js

```diff
@@ -5,25 +5,23 @@ import './App.css';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './PostList';
+
+const getPost = (postsArr, usersArr, commentArr) => (
+  postsArr.map((post) => {
+    const user = usersArr.find(person => person.id === post.userId);
+    const comment = commentArr
```

Comment:
```suggestion
    const comments = commentArr
```
Это же не один комментарий